app/Photo.php

```diff
@@ -170,13 +170,13 @@ public function prepareData()
 		$photo = array();
 
 		// Set unchanged attributes
-		$photo['id'] = $this->id;
+		$photo['id'] = strval($this->id);
 		$photo['title'] = $this->title;
 		$photo['tags'] = $this->tags;
 		$photo['star'] = $this->star == 1 ? '1' : '0';
-		$photo['album'] = $this->album_id;
-		$photo['width'] = $this->width;
-		$photo['height'] = $this->height;
+		$photo['album'] = $this->album_id !== null ? strval($this->album_id) : null;
+		$photo['width'] = strval($this->width);
+		$photo['height'] = strval($this->height);
```

Comment:
given that they are used as strings in the front-end sure. :)
Does this interfere with the Justified Layout ??